### ISSUE TITLE
Pool Scale is in the wrong place in focus order

### ISSUE THREAD

**Issue by Nancy-Salpepi** [phetsims/density-buoyancy-common#191]
**Test device**
MacBook Air M1 chip

**Operating System**
14.5

**Browser**
Safari 17.5

**Problem description**
For https://github.com/phetsims/qa/issues/1095:
On all screens the Pool Scale control is _after_ the Reset All button in the focus order. According to https://docs.google.com/document/d/1p-GwOR0jEsaJEouib6drVwLAti2J21-v1uVSpkoeZ5c/edit#heading=h.nhqxjbby3dgu from https://github.com/phetsims/density-buoyancy-common/issues/121 the pool scale should come after the other scale on the Compare, Explore and Shapes Screens. On the Lab Screen, it should be after the Block and on the Applications Screen after the Fluid Density control. 

<details>
Troubleshooting information:
!!!!! DO NOT EDIT !!!!!
Name: ‪Buoyancy‬
URL: https://phet-dev.colorado.edu/html/buoyancy/1.0.0-dev.9/phet/buoyancy_all_phet.html
Version: 1.0.0-dev.9 2024-06-12 15:13:18 UTC
Features missing: applicationcache, applicationcache, touch
Flags: pixelRatioScaling
User Agent: Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/605.1.15 (KHTML, like Gecko) Version/17.5 Safari/605.1.15
Language: en-US
Window: 1380x706
Pixel Ratio: 2/1
WebGL: WebGL 1.0
GLSL: WebGL GLSL ES 1.0 (1.0)
Vendor: WebKit (WebKit WebGL)
Vertex: attribs: 16 varying: 30 uniform: 1024
Texture: size: 16384 imageUnits: 16 (vertex: 16, combined: 32)
Max viewport: 16384x16384
OES_texture_float: true
Dependencies JSON: {}
</details>

**Comment by AgustinVallejo** [comment]
Assigning myself to work on this

**Comment by AgustinVallejo** [comment]
Addressed in the above commit. Please review and close if ready @Nancy-Salpepi. Thanks!

**Comment by Nancy-Salpepi** [comment]
Looks good!
Closing.
